airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -403,7 +403,7 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
         Returns next_page_token based on `end_of_stream` parameter inside of response
         """
         response_json = response.json()
-        return None if response_json.get(END_OF_STREAM_KEY, False) else {"start_time": response_json.get("end_time")}
+        return None if response_json.get(END_OF_STREAM_KEY, True) else {"start_time": response_json.get("end_time")}
```

Comment:
To confirm, this was the bug. The lack of an `end_time` is an indicator that the stream is complete?

Reply:
Yes, this the place, problem was in "infinite" pagination in case or error/wrong response

Reply:
thanks for confirming